mbta-performance/chalicelib/lamp/backfill/main.py

```diff
@@ -9,18 +10,19 @@
 def backfill_all_in_index():
     """Backfill all the dates in the LAMP index."""
 
-    # Load the index
-    index = pd.read_csv(LAMP_INDEX_URL)
-    # Get the dates in the index
-    dates = pd.to_datetime(index["service_date"]).dt.date
-    # Backfill each date
-    for date_to_backfill in dates:
+    # all dates that LAMP has data for, starting from 2019-09-15
+    dates = pd.date_range(datetime(2019, 9, 15).date(), datetime.today().date() - timedelta(days=1), freq="d")
```

Comment:
nit: can we define `EARLIEST_LAMP_DATA = datetime.date(2019, 9, 15)` at the beginning of the file?